display_books_promise.js

```diff
@@ -0,0 +1,66 @@
+let BookData = [];
```

Comment:
This is usually a way to name the class. Variable name should be - `let bookData = [];`